src/source/PeerConnection/SessionDescription.c

```diff
@@ -428,9 +428,10 @@ STATUS populateSingleMediaSection(PKvsPeerConnection pKvsPeerConnection, PKvsRtp
         CHK_STATUS(hashTableGet(pUnknownCodecPayloadTypesTable, unknownCodecHashTableKey, &payloadType));
     } else {
         CHK_STATUS(hashTableGet(pKvsPeerConnection->pCodecTable, pRtcMediaStreamTrack->codec, &payloadType));
-        currentFmtp = fmtpForPayloadType(payloadType, &(pKvsPeerConnection->remoteSessionDescription));
+        if (!pKvsPeerConnection->isOffer) {
+            currentFmtp = fmtpForPayloadType(payloadType, pRemoteSessionDescription);
```

Comment:
is non-null check handled?

Reply:
We actually can't because fmtpForPayloadType doesn't return a STATUS. We can add the null check here though.

Reply:
The null check is already present on line 1059. I will add another one here for redundancy.

Reply:
It seems like a lot of NULL checks are missing for these internal methods like `fmtpForPayloadType` (these methods are not publicly accessible). I'm going to timebox adding more null checks and we can take that up in a future PR.